src/lib.rs

```diff
@@ -576,7 +576,9 @@ extern crate pretty_assertions;
 #[macro_use]
 extern crate serde_derive;
 
+pub use oauth2::devicecode::{DeviceAuthorizationResponse, EmptyExtraDeviceAuthorizationFields};
```

Comment:
we should also re-export `DeviceCodeErrorResponseType`, `ExtraDeviceAuthorizationFields`, and `DeviceCodeErrorResponse` (see https://docs.rs/oauth2/4.4.0/oauth2/devicecode/index.html)